src/pages/SwapV3/index.tsx

```diff
@@ -1,346 +1,365 @@
+
+  const tradeRouteComposition = useMemo(() => {
+    return getTradeComposition(chainId, routeSummary?.parsedAmountIn, undefined, routeSummary?.route, defaultTokens)
+  }, [chainId, defaultTokens, routeSummary])
+
+  return (
+    <>
+      {isSwapPage && (
+        <>
+          <SEOSwap canonicalUrl={canonicalUrl} />
+          <TutorialSwap />
+        </>
+      )}
+      <TokenWarningModal
+        isOpen={isShowModalImportToken}
+        tokens={importTokensNotInDefault}
+        onConfirm={handleConfirmTokenWarning}
+      />
+      <PageWrapper>
+        <Banner />
+        <Container>
+          <SwapFormWrapper isShowTutorial={isShowTutorial}>
+            <Header activeTab={activeTab} setActiveTab={setActiveTab} />
+
+            {(isLimitPage || isSwapPage) && (
+              <PairSuggestion
+                ref={refSuggestPair}
+                onSelectSuggestedPair={onSelectSuggestedPair}
+                setShowModalImportToken={setShowingPairSuggestionImport}
+              />
+            )}
+
+            <AppBodyWrapped data-highlight={shouldHighlightSwapBox} id={TutorialIds.SWAP_FORM}>
+              {activeTab === TAB.SWAP && (
+                <PopulatedSwapForm
+                  onSelectSuggestedPair={onSelectSuggestedPair}
+                  routeSummary={routeSummary}
+                  setRouteSummary={setRouteSummary}
+                  goToSettingsView={() => setActiveTab(TAB.SETTINGS)}
+                  hidden={activeTab !== TAB.SWAP}
+                />
+              )}
+              {activeTab === TAB.INFO && <TokenInfoTab currencies={currencies} onBack={onBackToSwapTab} />}
+              {activeTab === TAB.SETTINGS && (
+                <SettingsPanel
+                  isCrossChainPage={isCrossChainPage}
+                  isLimitOrder={isLimitPage}
+                  isSwapPage={isSwapPage}
+                  onBack={onBackToSwapTab}
+                  onClickLiquiditySources={() => setActiveTab(TAB.LIQUIDITY_SOURCES)}
+                  onClickGasPriceTracker={() => setActiveTab(TAB.GAS_PRICE_TRACKER)}
+                />
+              )}
+              {activeTab === TAB.GAS_PRICE_TRACKER && (
+                <GasPriceTrackerPanel onBack={() => setActiveTab(TAB.SETTINGS)} />
+              )}
+              {activeTab === TAB.LIQUIDITY_SOURCES && (
+                <LiquiditySourcesPanel onBack={() => setActiveTab(TAB.SETTINGS)} />
+              )}
+              {activeTab === TAB.LIMIT && (
+                <LimitOrder
+                  isSelectCurrencyManual={isSelectCurrencyManually}
+                  setIsSelectCurrencyManual={setIsSelectCurrencyManually}
+                  refreshListOrder={refreshListOrder}
+                />
+              )}
+              {activeTab === TAB.CROSS_CHAIN && <CrossChain />}
+            </AppBodyWrapped>
+          </SwapFormWrapper>
+
+          {(isShowLiveChart || isShowTradeRoutes || shouldRenderTokenInfo || isLimitPage) && (

```

Comment:
remove  `isShowLiveChart || isShowTradeRoutes || shouldRenderTokenInfo || isLimitPage`
It is already check here
<img width="831" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/c38a23fd-6859-4c8a-b21a-b7741b691885">